src/lib/components/Accordion/Accordion.tsx

```diff
@@ -37,7 +38,7 @@ const AccordionComponent: FC<AccordionProps> = ({
 
   return (
     <div
-      className={classNames(theme.base, theme.flush[flush ? 'on' : 'off'])}
+      className={classNames(theme.base, className, theme.flush[flush ? 'on' : 'off'])}
```

Comment:
I guess that it would be better to have the `className` as last `classNames` param.

```jsx
className={classNames(theme.base, theme.flush[flush ? 'on' : 'off'], className)}
```